WcaOnRails/db/structure.sql

```diff
@@ -884,159 +724,119 @@ CREATE TABLE `eligible_country_iso2s_for_championship` (
   `eligible_country_iso2` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL,
   PRIMARY KEY (`id`),
   UNIQUE KEY `index_eligible_iso2s_for_championship_on_type_and_country_iso2` (`championship_type`,`eligible_country_iso2`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+) ENGINE=InnoDB AUTO_INCREMENT=9 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
-
---
--- Table structure for table `linkings`
---
-
 DROP TABLE IF EXISTS `linkings`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `linkings` (
-  `wca_id` varchar(10) NOT NULL,
-  `wca_ids` mediumtext NOT NULL,
+  `wca_id` varchar(10) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `wca_ids` mediumtext COLLATE utf8mb4_unicode_ci,
```

Comment:
Is it possible your local database is wrong? This migration makes it look like `wca_ids` should in fact be `NOT NULL`:

https://github.com/jfly/worldcubeassociation.org/blob/f834d079ef8db91cca2787c34d4c6b4effb0743b/WcaOnRails/db/migrate/20170426145811_create_linkings.rb#L7